xwiki-platform-core/xwiki-platform-livedata/xwiki-platform-livedata-webjar/src/main/webjar/layouts/livedata-table.html

```diff
@@ -0,0 +1,90 @@
+<template>
+    <table class="xwiki-livetable">
+
+        <thead>
+
+            <tr>
+
+                <th
+                    v-for="(col, cIdx) in cols"
+                    :key="cIdx"
+                >
+                    <div class="column-header">
+                        <div>{{ col.name }}</div>
+                    </div>
+                </th>
+
+            </tr>
+
+        </thead>
+
+
+
+        <tbody>
+
+            <tr
+                v-for="(row, rIdx) in rows"
+                :key="rIdx"
+            >
+
+                <td
+                    v-for="col in cols"
+                    :key="col.key"
+                >
+                    <div
+                        class="cell"
+                        v-text="row[col.key]"
+                    ></div>
```

Comment:
Why not put the ``cell`` class on the TD? I think it would make more sense. And why do you need the extra DIV wrapping? I wouldn't complicate the HTML structure unless it's really needed.

Reply:
As html table element are not very flexible I preferred to create a div element inside the td to avoid troubles in the future. But indeed, as it's not an issue for now I removed it.